code/components/citizen-devtools/src/ResourceTimeWarnings.cpp

```diff
@@ -12,6 +12,8 @@
 
 #include <chrono>
 
+#include <CfxLocale.h>

```

Comment:
When we revive localization, we may want to make this localizable. @technetium-cfx are you still able to look into configuring Weblate or finding an alternative solution with similar Git support?